.github/workflows/auto-tag-latest.yml

```diff
@@ -1,13 +1,16 @@
 name: "auto-tag-latest"
+permissions:
+  contents: write
 on:
   release:
     types: [published]
 jobs:
+  # move the "latest" and major version tags to the latest release, e.g., v1, to enable
```

Comment:
thank you for this PR, what did you mean by "to enable" in this comment ?

Reply:
I meant to document the purpose of this job: to update the "latest" and major version tags. This is so that consumers may choose between the latest version and the most recent reverse-compatible version (major version).